Enable milliseconds in syslog-ng for all pot log timestamps

thrashwerk opened the pull request:
Hello, this PR increases log timestamp accuracy by adding 3 fractional digits for seconds (a.k.a. milliseconds) for all pots that have syslog-ng configs.

Example timestamp without this change:
```
2024-10-27T06:11:12+00:00
```

Example timestamp with this setting:
```
2024-10-31T14:16:48.762+00:00
```

thrashwerk committed:
Enable milliseconds in syslog-ng for all pot log timestamps

bretton commented on tmp.sh:
for CHANGELOG.md I've been grouping multiple items under X.Y and then incrementing version x.y.Z in ini file.

there shouldn't be a tmp.sh in the repo

bretton committed:
Update influxdb.ini

not building this image currently, no need to increment ini file

bretton committed:
Update haproxy.ini

not building this image currently, no need to increment ini file

bretton committed:
Update CHANGELOG.md

bretton committed:
Update CHANGELOG.md

not building this image currently, not incrementing version

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton committed:
Update CHANGELOG.md

consistency, add bunch of changes under X.Y in CHANGELOG, with version increment x.y.Z in ini file

bretton commented:
thanks @thrashwerk 

some of these I don't want to rebuild images for, so the version number doesn't need an increment.

* haproxy
* influxdb

the CHANGELOG format was changed for a more condensed style some time back, with x.y.Z changes grouped under a single X.Y item. Some haven't adapted yet. 

Please remove the tmp.sh file

thrashwerk committed:
remove temp script
